usaspending_api/awards/v2/data_layer/orm.py

```diff
@@ -233,15 +238,20 @@ def create_place_of_performance_object(db_row_dict):
     )
 
 
-def create_officers_object(db_row_dict):
+def create_officers_object(award, mapper, transaction_type):
+
+    transaction = fetch_latest_ec_details(award["id"], mapper, transaction_type)
```

Comment:
If the only field necessary is the award.id field, would it be better for the parameter to be the award PK value like `fetch_fabs_details_by_pk()`?

Reply:
It's very subjective, but I would argue that taking in the whole award object follows single responsibility principle a little better by abstracting the exact logic to deciding what transaction maps to what award. On a practical level though I don't think it matters. I'm happy to change it for consistency.